resource/opentelemetry-resource-detector-container/README.rst

```diff
@@ -0,0 +1,47 @@
+OpenTelemetry Resource detectors for containers
+==========================================================
+
+|pypi|
+
+.. |pypi| image:: TODO
+   :target: TODO
+
+
+This library provides container property (container.id) detection features which can help
+in identifying the problems in app - infra correlation for apps running on containers i.e. docker
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-resource-detector-containerß
```

Comment:
```suggestion
    pip install opentelemetry-resource-detector-container
```

Reply:
@sanketmehta28 please review this comment, I am almost ready to approve this PR.